Assets/Decontainment/UI/Editors/Utilities/Tips/DisplaysTips.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Editor.Code;
+using Asm;
+
+namespace Editor
+{
+    public class DisplaysTips : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
+    {
+
+        public GameObject toolTipContainer;
+        public int tipDelay;
+
+        private GameObject toolTipInstance;
+        private GameObject canvas;
+        private int t;
+        private bool hovered;
+        private bool isField;
+        private string tipText;
+
+
+
+        void Awake()
+        {
+            canvas = GameObject.Find("Canvas");
+            isField = (GetComponent<SlotField>() != null
+                             || GetComponent<DropdownField>() != null) 
+                             ? true : false;
+        }
+
+        public void Init(string text)
+        {
+            tipText = text;
+        }
+
+        void Update()
+        {
+            if (hovered)
+            {
+                if ((toolTipInstance == null) && (t > tipDelay))
+                {
+                    if (tipText != null)
+                    {
+                        InstantiateTip();
+                    }
+                } else
+                {
```

Comment:
This bracket should be on the same line as the else statement. See style guide: https://docs.google.com/document/d/1VPMxvzhVjg1bco6NsSNZ46y_ja8B7wJTUkdx2zS7dQ4/edit?usp=sharing